### PR TITLE
Ch 17 backend models

### PR DESCRIPTION
**Pull request by amistein**

### CODE REVIEW

File: db/models/interview.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const db = require('APP/db');
+
+const Interview = db.define('interviews', {
+  date: {
+    type: Sequelize.DATE,
+    allowNull: false
+  },
+  position: Sequelize.STRING,
+  status: Sequelize.ENUM('planned', 'done'),
+  candidateRating: {
```
Comment (runandrew) [inline db/models/interview.js]: maybe we should name this `candidateOverallRating`? possibly to distinguish it from `candidateRating` from the join table. for our own sake, shouldn't matter in the execution

Comment (amistein) [reply db/models/interview.js]: sounds good to me

File: db/models/problem.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const db = require('APP/db');
+
+const Problem = db.define('problems', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false, 
+    unique: true
```
Comment (runandrew) [inline db/models/problem.js]: would a unique here cause problems? could multiple companies have a problem called `Fibonacci Sequence`? just thinking out loud...

Comment (amistein) [reply db/models/problem.js]: that's a good point. I think it should be changed.

File: server/api.js [hunk 3]
```diff
@@ -4,7 +4,13 @@ const db = require('APP/db')
...
 
 api
-  .get('/heartbeat', (req, res) => res.send({ok: true,}))
+  .get('/heartbeat', (req, res) => res.send({ok: true}))
+  // DELETE THIS CODE BEFORE MERGE
+  // .get('/testDate', (req, res) => {
+  //   db.model('interviews').findAll({})
+  //   .then(interview => interview[0].date)
+  //   .then(date => res.send(date));
+  // })
```
Comment (runandrew) [inline server/api.js]: Should we leave these comments in? says delete before merge

**Review by runandrew — CHANGES_REQUESTED** [review]
looks awesome overall!

**Review by runandrew — APPROVED** [review]

**Pull request merged**